ui/app/components/app-form/project-settings.hbs

```diff
@@ -0,0 +1,163 @@
+<form class="pds-form">
+  <div part="fields">
+    <fieldset class="pds-formFieldSet">
+      <div class="pds-formField">
+        <label for="git-source-url" class="pds-fieldName">
+          Git Source URL
+        </label>
+        <Pds::Input
+          @type="text"
+          id="git-source-url"
+          placeholder="git@github.com:mycompany/awesome-project.git"
+          value={{this.git.url}}
+          required
+          {{on "input" (set this "project.dataSource.git.url" (get _ "target.value"))}}
```

Comment:
I noticed this in HCP a little bit ago - I think this syntax is pre 1.0 of the set helper 😭 - just something to keep an eye out for!

Reply:
Ah I saw you're requiring this explicitly - it looks like the 1.0 one they suggest using set with `pick` from `ember-composable-helpers`: https://github.com/pzuraq/ember-set-helper#picking-values-with-ember-composable-helpers - I'd recommend going that route and with the most recent release so you're not stuck with an upgrade later on.

Reply:
re: `pick`, yeah I saw that and hesitated, but instead mimicked what you had, to then realize I had to pin a specific version of the `get` helper 😅 . Good point, I think i'll update that so we start with the right patterns

Reply:
done: https://github.com/hashicorp/waypoint/pull/1057/commits/6069d6b798455f60db8ddc7a702a7d670c52af2e